runtime/binding-http-kafka/src/main/java/io/aklivity/zilla/runtime/binding/http/kafka/internal/config/HttpKafkaWithProduceResult.java

```diff
@@ -83,7 +84,8 @@ public class HttpKafkaWithProduceResult
         String16FW idempotencyKey,
         List<HttpKafkaWithProduceAsyncHeaderResult> async,
         HttpKafkaWithProduceHash hash,
-        long timeout)
+        long timeout,
+        String16FW correlationId)
```

Comment:
It seems to make sense to move this up near `replyTo` as they are logically related parameters.